src/main/java/edu/ucsb/cs56/ucsb_courses_search/model/search/SearchByGEMultiQuarter.java

```diff
@@ -0,0 +1,79 @@
+package edu.ucsb.cs56.ucsb_courses_search.model.search;
+
+import java.util.Objects;
+
+public class SearchByGEMultiQuarter {
+    private String college;
+    private String area;
+    private int beginQ;
+    private int endQ;
+
+    public SearchByGEMultiQuarter() { } 
+
+    public SearchByGEMultiQuarter(String college, String area, String year, int beginQ, int endQ) {
+        this.college = college;
+        this.area = area;
+        this.beginQ = beginQ;
+        this.endQ = endQ;
+    }
+
+    public String getCollege() {
+        return this.college;
+    }
+
+    public void setCollege(String college) {
+        this.college = college;
+    }
+
+    public String getArea() {
+        return this.area;
+    }
+
+    public void setArea(String area) {
+        this.area = area;
+    }
+
+    public int getBeginQ() {
+        return this.beginQ;
+    }
+
+    public void setBeginQ(int beginQ) {
+        this.beginQ = beginQ;
+    }
+
+    public int getEndQ() {
+        return this.endQ;
+    }
+
+    public void setEndQ(int endQ) {
+        this.endQ = endQ;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this)
+            return true;
+        if (!(o instanceof SearchByGEMultiQuarter)) {
+            return false;
+        }
+        SearchByGEMultiQuarter SearchByGEMultiQuarter = (SearchByGEMultiQuarter) o;
+        return Objects.equals(college, SearchByGEMultiQuarter.college) && 
+                              Objects.equals(area, SearchByGEMultiQuarter.area) && beginQ == SearchByGEMultiQuarter.beginQ && endQ == SearchByGEMultiQuarter.endQ ;
```

Comment:
You might want to rename the variable name, so that it's proper lowerCamelCase. The naming you have right now makes it a little hard to understand.

You shouldn't run into this issue if you auto-generate your equals/hashCode methods.

```suggestion
        SearchByGEMultiQuarter searchByGEMultiQuarter = (SearchByGEMultiQuarter) o;
        return Objects.equals(college, searchByGEMultiQuarter.college) && 
                              Objects.equals(area, searchByGEMultiQuarter.area) && beginQ == searchByGEMultiQuarter.beginQ && endQ == searchByGEMultiQuarter.endQ ;
```